### ISSUE TITLE
ModuleNotFoundError: No module named 'utils'

### ISSUE THREAD

**Issue by Daydreamer-w** [wuwusky/RobustDeepLeraningRegistration#2]

**Comment by wuwusky** [comment]
Thank you for the reminder, the utils.py file has been supplemented.

**Comment by Daydreamer-w** [comment]
> Thank you for the reminder, the utils.py file has been supplemented.

Thank you for your reply, but the error still persists.@wuwusky

`ImportError: cannot import name 'get_img_pad' from 'utils'`

**Comment by wuwusky** [comment]
> > Thank you for the reminder, the utils.py file has been supplemented.
> 
> Thank you for your reply, but the error still persists.@wuwusky
> 
> `ImportError: cannot import name 'get_img_pad' from 'utils'`

You are correct, and I appreciate the update to the corresponding utils.py and utils_eff.py files, as well as the upload of the training code train_RM. I hope these changes and additions are of significant assistance for you.

**Comment by Daydreamer-w** [comment]
> > > Thank you for the reminder, the utils.py file has been supplemented.
> > 
> > 
> > Thank you for your reply, but the error still persists.@wuwusky
> > `ImportError: cannot import name 'get_img_pad' from 'utils'`
> 
> You are correct, and I appreciate the update to the corresponding utils.py and utils_eff.py files, as well as the upload of the training code train_RM. I hope these changes and additions are of significant assistance for you.

Thank you for your update. Do you have any plans to publicly disclose available weights in ROP?@wuwusky

**Comment by wuwusky** [comment]
> > > > Thank you for the reminder, the utils.py file has been supplemented.
> > > 
> > > 
> > > Thank you for your reply, but the error still persists.[@wuwusky](https://github.com/wuwusky)
> > > `ImportError: cannot import name 'get_img_pad' from 'utils'`
> > 
> > 
> > You are correct, and I appreciate the update to the corresponding utils.py and utils_eff.py files, as well as the upload of the training code train_RM. I hope these changes and additions are of significant assistance for you.
> 
> Thank you for your update. Do you have any plans to publicly disclose available weights in ROP?[@wuwusky](https://github.com/wuwusky)

model weights have been uploaded, enjoy~
